test/functional/kernel/hyperplane/test-kernel-hyperplane-3D.cpp.in

```diff
@@ -193,6 +193,16 @@ using HipKernelHyperplaneExecPols =
 // no existing SYCL implementation of Hyperplane
 #endif  // RAJA_ENABLE_SYCL
 
+#define @REDUCER_TYPE@
+// Switch to control testing old vs new reducer interface
+#if defined(ParamReduce) and not defined(RAJA_ENABLE_SYCL)
+using UseParamReducers = camp::list<std::true_type>;
+#elif defined(CaptureReduce)
+using UseParamReducers = camp::list<std::false_type>;
+#else
+static_assert(false, "unreachable");
+#endif
+
```

Comment:
You could make this the same as the others where they are all defined but one is picked based on the configuration.
Maybe something like this with `Use@REDUCER_TYPE@` below?
```suggestion
using UseParamReduce = camp::list<std::true_type>;

using UseCaptureReduce = camp::list<std::false_type>;
```